apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Flex, IconsLegacy, Text } from "@ledgerhq/react-ui";
+import { Trans } from "react-i18next";
+import { useTheme } from "styled-components";
+import { rgba } from "~/renderer/styles/helpers";
+
+type Size = "small" | "medium" | "large";
+
+export type DesignedForStaxTextProps = {
+  size?: Size;
+  hideBackground?: boolean;
+};
+
+const LedgerLogo = ({
+  // children isn't used at all but it's easier to maintain translation files
+  // if the logo can be manipulated as a "Ledger" string
+  children: _,
+  size,
+  hideBackground,
+}: {
+  children?: string;
+} & DesignedForStaxTextProps) => {
+  const { colors } = useTheme();
+  return (
+    <Flex px={size === "small" ? 1 : 2}>
+      <IconsLegacy.LedgerLogoMedium
+        size={size === "medium" ? 24 : size === "large" ? 36 : 16}
```

Comment:
We have different font sizes used here, can't we use the ones from our theme? Is there any Figma that defines these font size values?